build.gradle

```diff
@@ -460,6 +541,13 @@ subprojects {
             url gradleRemoteRepositoryUrl
         }
     }
+    // disable oomph projects on targets without oomph
+    if (['43', '44'].contains(eclipsebuild.Config.on(project).targetPlatform.eclipseVersion) && name.startsWith("org.eclipse.buildship.oomph")) {
```

Comment:
Sure, we can tie the generation to a project property. Since oomph only needs to be disabled for two of the target platforms I propose defaulting the switch to true.

Reply:
👍